iocore/cache/CacheWrite.cc

```diff
@@ -102,10 +102,12 @@ CacheVC::updateVector(int /* event ATS_UNUSED */, Event * /* e ATS_UNUSED */)
     }
     if (vec) {
       /* preserve fragment offset data from old info. This method is
-         called iff the update is a header only update so the fragment
+         called if the update is a header only update so the fragment
```

Comment:
"iff" is correct - it's a standard abbreviation for "if and only if".

Reply:
Thanks to correct me. will revert.